File: cs/ViewModels/MyViewModelBase.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.DataAnnotations;

namespace DXSample.ViewModels {
    public class MyViewModelBase : ViewModelBase {
        public string Caption {
            get { return GetProperty(() => Caption); }
            set { SetProperty(() => Caption, value); }
        }
        [Command]
        public void ChangeText() {
            (((ISupportParentViewModel)this).ParentViewModel as MainViewModel).SplashScreenViewModel.Caption = this.Caption;
        }
    }
}

File: VB/ViewModels/MyViewModelBase.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.DataAnnotations

Namespace DXSample.ViewModels

    Public Class MyViewModelBase
        Inherits ViewModelBase

        Public Property Caption As String
            Get
                Return GetProperty(Function() Me.Caption)
            End Get

            Set(ByVal value As String)
                SetProperty(Function() Caption, value)
            End Set
        End Property

        <Command>
        Public Sub ChangeText()
            TryCast(CType(Me, ISupportParentViewModel).ParentViewModel, MainViewModel).SplashScreenViewModel.Caption = Caption
        End Sub
    End Class
End Namespace
